Layout-with-Position/css-positioning_layout-with-position/css/styles.css

```diff
@@ -48,15 +60,21 @@ body {
  * Define the card as reference frame for descendant elements, that require absolute positioning.
  */
 .card {
+
```

Comment:
You need to define this class with position: relative, so all the child-elements can refer to it.